deepcell/utils/data_utils.py

```diff
@@ -507,3 +509,28 @@ def reshape_movie(X, y, reshape_size=256):
     print('Reshaped feature data from {} to {}'.format(y.shape, new_y.shape))
     print('Reshaped training data from {} to {}'.format(X.shape, new_X.shape))
     return new_X, new_y
+
+
+def rotation_mat2D(data):
+    """apply 2D roation"""
+    assert data.shape[-1] == 2, 'data is not 2D'
+    theta = np.radians(np.random.randint(0,360))
+    rot = np.array([[np.cos(theta), -np.sin(theta)],
+                    [np.sin(theta), np.cos(theta)]])
+    
+    return tf.matmul(data,rot)
```

Comment:
Could this be `np.matmul` instead? Mixing the np.arrays with tensorflow makes me a bit nervous.